src/vorta/scheduler.py

```diff
@@ -15,15 +18,95 @@
 logger = logging.getLogger(__name__)
 
 
+def trigger_equal(trig1, trig2):
+    if not isinstance(trig1, type(trig2)):
+        return False
+
+    if hasattr(trig1, 'run_date'):
+        # DateTrigger
+        return trig1.run_date == trig2.run_date
+    elif hasattr(trig1, 'interval_length'):
+        # IntervalTrigger
+        return trig1.interval_length == trig2.interval_length
+    elif hasattr(trig1, 'fields'):
+        # CronTrigger
+        return trig1.fields == trig2.fields
+    else:
+        # unhandled trigger type
+        raise NotImplementedError
+
+
 class VortaScheduler(QtScheduler):
+    failed_profiles = set()
+
     def __init__(self, parent):
         super().__init__()
         self.app = parent
+
+        # persist jobs to database to continue schedule of relative interval jobs
+        self.configure(jobstores={'default': SQLAlchemyJobStore(url=f'sqlite:///{DB_PATH}')})
```

Comment:
It's not a problem if 2 ORMs use the same database file?

Reply:
They are using different tables:
```
sqlite> .tables
apscheduler_jobs    eventlogmodel       schemaversion       sourcedirmodel    
archivemodel        repomodel           settingsmodel       wifisettingmodel  
backupprofilemodel  repopassword        snapshotmodel     
```

However, it would also be fine to have put the jobs into a separate database. Right now, I don't see a strong reason either for or against it. So, your call ;)